cmd/xl-storage.go

```diff
@@ -2385,36 +2419,12 @@ func (s *xlStorage) CheckParts(ctx context.Context, volume string, path string,
 	}
 
 	for i, part := range fi.Parts {
-		partPath := pathJoin(path, fi.DataDir, fmt.Sprintf("part.%d", part.Number))
-		filePath := pathJoin(volumeDir, partPath)
-		st, err := Lstat(filePath)
+		resp.Results[i], err = xioutil.WithDeadline[int](ctx, globalDriveConfig.GetMaxTimeout(), func(ctx context.Context) (int, error) {
```

Comment:
What is the default drive timeout, 1m?

Reply:
no, it is 30 seconds

Reply:
ok. But it never reads through the part - that is done in other checks, correct? 

If so, I am fine with the deadline and you can just resolve this.

Reply:
>  But it never reads through the part 

yes, it never reads the part. This is the light healing scan where we do not do bitrot check